### PR TITLE
Make Sidestore use its own trustedapps json

### PR DESCRIPTION
**Pull request by Spidy123222**
This adds and forces sidestore to use a trustedapps.json file for list of trusted sources.

### CODE REVIEW

File: AltStore/Operations/FetchTrustedSourcesOperation.swift [hunk 1]
```diff
@@ -11,9 +11,9 @@ import Foundation
 private extension URL
 {
     #if STAGING
-    static let trustedSources = URL(string: "https://f000.backblazeb2.com/file/altstore-staging/altstore/trustedsources.json")!
+    static let trustedSources = URL(string: "https://raw.githubusercontent.com/Spidy123222/SideStore/change-source/trustedapps.json")!
```
Comment (jkcoxson) [inline AltStore/Operations/FetchTrustedSourcesOperation.swift:14]: This still says Spidy’s repo

Comment (Spidy123222) [reply AltStore/Operations/FetchTrustedSourcesOperation.swift:14]: This is currently for testing. Once confirmed it works then can change it to sidestore.

Comment (Spidy123222) [reply AltStore/Operations/FetchTrustedSourcesOperation.swift:14]: > This still says Spidy’s repo

I have been able to confirm it works I’ll make necessary changes.

Comment (Spidy123222) [reply AltStore/Operations/FetchTrustedSourcesOperation.swift]: Changes are now directed to Sidestore Repo

**Comment by Spidy123222** [comment]
I have tested this myself and has worked perfectly fine. It’s matter of should it be in the main repo or a separate one like the default apps could.

**Review by JJTech0130 — APPROVED** [review]
It looks good, but what is the io.altstore.example line?

**Comment by Spidy123222** [comment]
> It looks good, but what is the io.altstore.example line?

This is what came up when I put the json they were using on a web browser. Idk why they have it though so I just didn't touch it.

**Pull request merged**